networks/local/localnode/Dockerfile

```diff
@@ -13,4 +13,4 @@ CMD ["node", "--proxy_app", "kvstore"]
 STOPSIGNAL SIGTERM
 
 COPY wrapper.sh /usr/bin/wrapper.sh
-
+COPY config-template.toml /etc/tendermint/config-template.toml
```

Comment:
I added this here but to no effect (see the [CircleCI build logs](https://circleci.com/gh/tendermint/tendermint/61887?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)).

Upon investigation, it appears as though the `localnet` test pulls the `localnode` Docker image, which was [last updated 6 months ago](https://cloud.docker.com/u/tendermint/repository/docker/tendermint/localnode).

@greg-szabo: Is there somewhere where `localnode` gets built/pushed automatically? Or is it a manual process?

I'm going to make it a prerequisite for `localnet-start` to build `localnode` locally for now and see if that helps the build process.

Reply:
Seems like it did help the build process 😁